frontend/public/components/overview/project-overview.tsx

```diff
@@ -215,7 +215,7 @@ const ProjectOverviewListItem = connect<ProjectOverviewListItemPropsFromState, P
     // Hide metrics when a selection is active.
     const hasSelection = !!selectedUID;
     const isSelected = uid === selectedUID;
-    const className = classnames('project-overview__item', {'project-overview__item--selected': isSelected});
+    const className = classnames(`project-overview__item project-overview__item--${item.obj.kind.toLowerCase()}`, {'project-overview__item--selected': isSelected});
```

Comment:
Suggest just using `kind` as-is.

```suggestion
    const className = classnames('project-overview__item', `project-overview__item--${kind}`, {'project-overview__item--selected': isSelected});
```

Reply:
This would avoid runtime errors if `kind` is not defined and also any problems if `kind.toLowerCase()` doesn't match `model.id` (which is possible)